extensions/hide-my-mail-raycast/src/services/api/config.ts

```diff
@@ -0,0 +1,23 @@
+import { getPreferenceValues } from "@raycast/api";
+import { ApiConfig } from "../../types";
+
+export function getApiConfig(): ApiConfig {
+  const preferences = getPreferenceValues<{
+    cloudflareApiKey: string;
+    cloudflareAccountId: string;
+    cloudflareZoneId: string;
+    destinationEmail: string;
+    preAllocatePool: boolean;
+  }>();
```

Comment:
```suggestion
  const preferences = getPreferenceValues<Preferences>();
```

The `Preferences` type is auto-generated by Raycast so you can remove your own typing.